### PR TITLE
Add Asset Eligibility Formula

### PR DESCRIPTION
**Pull request by tolaouk**

### CODE REVIEW

File: openfisca_us/parameters/hhs/ccdf/asset_limit.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+description: CCDF asset limit
+
+values:
+  2021-10-01: 1_000_000
+
+metadata:
+  unit: currency-USD
+  reference:
+  - title: CCDF plan for NY - FFY 2022-2024
```
Comment (MaxGhenis) [inline openfisca_us/parameters/hhs/ccdf/asset_limit.yaml:9]: Could you cite a US-wide source?

**Review by MaxGhenis — APPROVED** [review]

**Comment by MaxGhenis** [comment]
Great, if you can just resolve the merge conflicts (sorry, my code reorganization one created them) we'll be good to merge.

File: openfisca_us/parameters/hhs/ccdf/asset_limit.yaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+description: CCDF asset limit
+
+values:
+  2021-10-01: 1_000_000
```
Comment (MaxGhenis) [inline openfisca_us/parameters/hhs/ccdf/asset_limit.yaml:4]: Per sources in https://github.com/PolicyEngine/openfisca-us/issues/325, this was not part of the 1990 act but was introduced in the 2014 act. So please set it to 0 effective 1991-01-01 and 1_000_000 effective 2015-01-01.

File: openfisca_us/tests/policy/baseline/demographic/household/ccdf_asset_eligibility.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+- name: Family asset, 1,000,000
+  period: 2022
+  input:
+    spm_unit_assets: 1_000_000
+  output:
+    is_ccdf_asset_eligible: True
+
+- name: Family asset, 1,000,001
```
Comment (MaxGhenis) [inline openfisca_us/tests/policy/baseline/demographic/household/ccdf_asset_eligibility.yaml:8]: Could you add a copy of this for 2014 to test that it's true?

**Review by nikhilwoodruff — APPROVED** [review]
Thanks @tolaouk - looks good to me.

**Pull request merged**